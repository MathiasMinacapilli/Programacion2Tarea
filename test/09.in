# pruebas para tabla 2
asociar_en_tabla 1 m
asociar_en_tabla 2 n
asociar_en_tabla 3 o
asociar_en_tabla 4 p
asociar_en_tabla 5 q
esta_llena_tabla
valor_en_tabla 1
valor_en_tabla 2
valor_en_tabla 3
valor_en_tabla 4
valor_en_tabla 5
existe_asociacion 1
existe_asociacion 2
existe_asociacion 3
existe_asociacion 4
existe_asociacion 5
asociar_en_tabla 1 mm
asociar_en_tabla 2 nn
asociar_en_tabla 3 oo
valor_en_tabla 1
valor_en_tabla 2
valor_en_tabla 3
valor_en_tabla 4
valor_en_tabla 5
valor_en_tabla 0
existe_asociacion 1
existe_asociacion 2
existe_asociacion 3
existe_asociacion 4
existe_asociacion 0
esta_llena_tabla
eliminar_de_tabla 0
eliminar_de_tabla 1
eliminar_de_tabla 2
eliminar_de_tabla 3
eliminar_de_tabla 4
esta_llena_tabla
eliminar_de_tabla 5
asociar_en_tabla 1 m
asociar_en_tabla 2 n
asociar_en_tabla 3 o
asociar_en_tabla 4 p
asociar_en_tabla 5 q
asociar_en_tabla 6 r
asociar_en_tabla 7 s
asociar_en_tabla 8 t
asociar_en_tabla 9 u
asociar_en_tabla 10 v
asociar_en_tabla 11 w
asociar_en_tabla 12 y
asociar_en_tabla 13 z
asociar_en_tabla 14 p
esta_llena_tabla
valor_en_tabla 1
valor_en_tabla 2
valor_en_tabla 3
valor_en_tabla 4
valor_en_tabla 5
valor_en_tabla 10
valor_en_tabla 11
valor_en_tabla 12
valor_en_tabla 13
valor_en_tabla 7
valor_en_tabla 6
valor_en_tabla 8
valor_en_tabla 9
valor_en_tabla 22
eliminar_de_tabla 1
eliminar_de_tabla 2
eliminar_de_tabla 3
eliminar_de_tabla 4
eliminar_de_tabla 5
eliminar_de_tabla 6
eliminar_de_tabla 7
eliminar_de_tabla 8
eliminar_de_tabla 9
eliminar_de_tabla 10
eliminar_de_tabla 11
eliminar_de_tabla 12
eliminar_de_tabla 13
eliminar_de_tabla 14
Fin
