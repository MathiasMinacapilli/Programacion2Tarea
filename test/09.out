1>#  pruebas para tabla 2.
2>Asociado.
3>Asociado.
4>Asociado.
5>Asociado.
6>Asociado.
7>t NO está llena.
8>m.
9>n.
10>o.
11>p.
12>q.
13>Existe asociación en t.
14>Existe asociación en t.
15>Existe asociación en t.
16>Existe asociación en t.
17>Existe asociación en t.
18>Asociado.
19>Asociado.
20>Asociado.
21>mm.
22>nn.
23>oo.
24>p.
25>q.
26>No existe asociación en t.
27>Existe asociación en t.
28>Existe asociación en t.
29>Existe asociación en t.
30>Existe asociación en t.
31>NO existe asociación en t.
32>t NO está llena.
33>No existe asociación en t.
34>Desasociado.
35>Desasociado.
36>Desasociado.
37>Desasociado.
38>t NO está llena.
39>Desasociado.
40>Asociado.
41>Asociado.
42>Asociado.
43>Asociado.
44>Asociado.
45>Asociado.
46>Asociado.
47>Asociado.
48>Asociado.
49>Asociado.
50>Asociado.
51>Asociado.
52>Asociado.
53>t esta llena.
54>t está llena.
55>m.
56>n.
57>o.
58>p.
59>q.
60>v.
61>w.
62>y.
63>z.
64>s.
65>r.
66>t.
67>u.
68>No existe asociación en t.
69>Desasociado.
70>Desasociado.
71>Desasociado.
72>Desasociado.
73>Desasociado.
74>Desasociado.
75>Desasociado.
76>Desasociado.
77>Desasociado.
78>Desasociado.
79>Desasociado.
80>Desasociado.
81>Desasociado.
82>No existe asociación en t.
83>Fin.
